### ISSUE TITLE
undefined method `casecmp' for nil:NilClass in lib/fontist/font.rb:124:in `check_and_confirm_required_license'

### ISSUE THREAD

**Issue by CAMOBAP** [fontist/fontist#202]
```
...
      # --- Caused by: ---
      # NoMethodError:
      #   undefined method `casecmp' for nil:NilClass
      #   ./vendor/bundle/ruby/2.6.0/gems/fontist-1.8.1/lib/fontist/font.rb:124:in `check_and_confirm_required_license'
```

Full logs
 https://github.com/metanorma/metanorma-ogc/pull/219/checks?check_run_id=1625767937

**Comment by alexeymorozov** [comment]
Thanks! The issue would be fixed in #204.

Though this error would disappear, there could be a `LicensingError`. My guess that the `agree` variable is set to false in https://github.com/metanorma/metanorma-standoc/blob/v1.7.2/lib/asciidoctor/standoc/converter.rb#L161

**Comment by alexeymorozov** [comment]
Released v1.8.2 🚀 

Let's try it
